src/Language/PureScript/Errors.hs

```diff
@@ -455,6 +477,11 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
       unknownInfo u = line $ markCode ("t" <> T.pack (show u)) <> " is an unknown type"
 
     renderSimpleErrorMessage :: SimpleErrorMessage -> Box.Box
+    renderSimpleErrorMessage (InternalCompilerError ctx err) =
+      paras [ line "Internal compiler error:"
```

Comment:
I think this error message should include the same "please report this as a bug" copy that `internalError` does, preferably.